packages/terra-icon/src/IconBase.jsx

```diff
@@ -37,7 +37,7 @@ const propTypes = {
   /**
    * Focusable attribute. IE 10/11 are focusable without this attribute.
    */
-  focusable: PropTypes.bool,
+  focusable: PropTypes.string,
```

Comment:
Why this change?

Reply:
focusable is actually a string node for SVGs (https://allyjs.io/tutorials/focusing-in-svg.html) and react was throwing prop type errors when we gave it a boolean instead of a string.